app/lib/jsonapi.js

```diff
@@ -471,7 +487,10 @@ jsonapi.routeModelPostObjectResource = function(app, Model) {
      * Executes any available post-POST routine available for Model
      */
     var executePostRoutine = (document, done) => {
+      debug('executePostRoutine... ', Model.collection.collectionName);
+
       if (Model.jsonapi.post && Model.jsonapi.post.post) {
+
```

Comment:
Please integrate https://eslint.org/docs/rules/padded-blocks with string option "never" to avoid line breaks such as this one.